.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
I think this is something we'd want set to true?

Reply:
This was just a default, but I'll set it to true.